examples/win-title

```diff
@@ -3,13 +3,18 @@
 # Example for $XDG_CONFIG_HOME/nsxiv/exec/win-title
 # Called by nsxiv(1) on each redraw.
 # The output is set as nsxiv's window title.
+#
 # Arguments, "Optional" arguments might be empty:
-#   $1: full path to the current image
+#   $1: resolved absolute path of the current file
```

Comment:
Btw we *can* use `image` here since the current item is bound to be a valid image or will be removed from the filelist. But I'm OK with leaving it as `file` as well for consistency.

Reply:
> But I'm OK with leaving it as `file` as well for consistency.

Yeah, that was my thought as well.